components/sankey/elements/title.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Title
+page_title: Sankey Title
+description: Title of the Sankey Diagram for Blazor.
+slug: sankey-title
+tags: telerik,blazor,sankey,diagram,chart,title
+published: True
+position: 7
+---
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+# Sankey Title
+
+The Sankey diagram for Blazor allows you to specify a title for the diagram. This article explains how to add and configure the title.
+
+## Adding a Title
+
+1. Declare a `<SankeyTitle>` tag as a direct child of `<TelerikSankey>`.
+1. Set the `Text` parameter of the `<SankeyTitle>` tag.
+
+## Customizing the Title
```

Comment:
This article uses too many headings with too little content. Why not remove this heading and use the same structure as the other element articles?